File: CompSci 1/Unit 4, Operators/Grades/Grades.java
public class Grades {
    public static void main(String[] args) {
        double dailyAvg = 88;
        double quizAvg = 74;
        double testAvg = 95;
        double sixWeeksAvg = 0;

        int roundedAvg = 0;

        sixWeeksAvg = (dailyAvg * 0.3) + (quizAvg * 0.2) + (testAvg * 0.5);

        roundedAvg = (int)(sixWeeksAvg + 0.5);

        System.out.println("Daily Average = " + dailyAvg);
        System.out.println("Quiz Average = " + quizAvg);
        System.out.println("Test Average = " + testAvg);
        System.out.println("--------------------------");
        System.out.println("Six Weeks Average = " + roundedAvg);
    }
}

File: CompSci 1/Unit 4, Operators/Grades/Grades.kt
fun main(args: Array<String>) {
    val dailyAvg = 88.0
    val quizAvg = 74.0
    val testAvg = 95.0
    var sixWeeksAvg = 0.0

    var roundedAvg = 0

    sixWeeksAvg = dailyAvg * 0.3 + quizAvg * 0.2 + testAvg * 0.5

    roundedAvg = (sixWeeksAvg + 0.5).toInt()

    println("Daily Average = " + dailyAvg)
    println("Quiz Average = " + quizAvg)
    println("Test Average = " + testAvg)
    println("--------------------------")
    println("Six Weeks Average = " + roundedAvg)
}